webpack.config.dev.js

```diff
@@ -86,7 +86,7 @@ module.exports = (env = {}) => {
         })
     });
 
-    COMPONENTS_DEV_CONFIG.entry = ['US', 'US-EZP', 'DE', 'GB'].reduce(
+    COMPONENTS_DEV_CONFIG.entry = ['US', 'US-EZP', 'DE', 'GB', 'FR'].reduce(
```

Comment:
Would it make sense at this point to have a shared config somewhere with a list of supported countries to pull from here and elsewhere?